### DISCUSSION TITLE
concurrency_visualizer报错

### DISCUSSION THREAD

**Discussion by oliver-zhang987** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#70] [Q&A]
![屏幕截图 2023-01-10 174726](https://user-images.githubusercontent.com/83524709/211517709-82a5f5fd-3632-48cc-94d1-989dca1ee406.jpg)

想问下 这个报错是因为什么呢？ concurrency前面的test都pass了

**Comment by TANSixu** [comment]
Hi，应该是simsocket没有更新，可能merge时出了点问题。可以尝试重新覆盖一下simsocket的代码，修改可参考 #33

**Reply by oliver-zhang987** [reply]
哦哦 覆盖之后解决了 谢谢！
那像这样的图应该可以证明实现了concurrency吧？不知道是覆盖了还是怎么，40083的点看不到几个
![屏幕截图 2023-01-10 201328](https://user-images.githubusercontent.com/83524709/211548962-0b2930fe-434b-4dbd-a9c1-08294c0330fc.jpg)

**Reply by TANSixu** [reply]
没问题，这是非常好的concurrency，很棒！
看不到是因为是因为离得太近了，覆盖了。可以把这个test对应的topo文件中两个链路的带宽和延迟调整一下，使差别大一些，concurrency的效果会在图上更明显噢～

**Reply by oliver-zhang987** [reply]
好的好的 谢谢
